wagtail/admin/rich_text/editors/hallo.py

```diff
@@ -102,11 +102,16 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
-    def render(self, name, value, attrs=None):
+    def translate_value(self, value):
+        # Convert database rich text representation to the format required by
+        # the input field
         if value is None:
```

Comment:
The condition could be reversed to remove a `return`. Or a ternary line.

Reply:
For methods where the return value is used, I think the clarity of an explicit `return None` is more important than saving a line of code.